### Chess piece message.

# piece enumeration constants
uint8 NoPiece = 48  # '0' (ascii zero)
uint8 King    = 75  # 'K'
uint8 Queen   = 81  # 'Q'
uint8 Rook    = 82  # 'R'
uint8 Bishop  = 66  # 'B'
uint8 Knight  = 78  # 'N'
uint8 Pawn    = 80  # 'P'

# the arguments
uint8 piece
